modules/402-ingress-nginx/images/controller-0-48/rootfs/etc/nginx/lua/pbmetrics.lua

```diff
@@ -71,18 +71,21 @@ message HistogramMessage {
     map<string, uint64> Buckets = 3;
     double Sum = 4;
     uint64 Count = 5;
+    string NamespacedIngress = 6;
```

Comment:
Oh, `CounterMessage`, `GaugeMessage` and `HistogramMessage` are all abstract. The `protobuf` exporter itself is also abstract, but now there is a kubernetes ingress related property.  Honestly, I'd like to avoid making this exporter less generic.

Reply:
+